pkgs/top-level/all-packages.nix

```diff
@@ -5581,6 +5581,20 @@ in
 
   srcml = callPackage ../applications/version-management/srcml { };
 
+  srhtPackages = callPackage ../applications/version-management/sr-ht {
+    # https://github.com/NixOS/nixpkgs/pull/54425#discussion_r249489825
+    pythonPackages = python36Packages;
```

Comment:
Better use `python36Packages` as argument in the expression and move the comment there.